scripts/check-konflux-pipelines.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
1. Please unify the script.
2. Please make sure the diff is printed so that the developer knows what to put in `runAfter`.

Reply:
See https://github.com/stackrox/stackrox/pull/13577#discussion_r1893773899